handbook/product/user_research/user_research_participant.md

```diff
@@ -0,0 +1,18 @@
+# So you’re about to participate in user testing
+
+Thank you so much for helping with our user research! Remote moderated hallway testing is a great way for us to quickly test and iterate on our ideas so that we can identify opportunities and challenges early.
+Here’s everything you need to know ahead of time.
+
+## Prerequisites
+
+- The tool we use, Lookback, works with Chrome. You’ll need to use Chrome to join the session.
```

Comment:
```suggestion
- The tool we use, [Lookback](https://lookback.io/), works with Chrome. You’ll need to use Chrome to join the session.
```

Reply:
@quinnkeast do participants need to log in?

Reply:
Nope!